AsyncDisplayKitTests/ASEditableTextNodeTests.m

```diff
@@ -0,0 +1,103 @@
+//
+//  ASEditableTextNodeTests.m
```

Comment:
Very nice to have this!  Previously this class was one of the most highlighted under-tested areas of the codebase so this will help the code coverage metric a lot.